tft_bot/helpers/system_helpers.py

```diff
@@ -74,25 +74,23 @@ def find_in_processes(executable_path: str) -> bool:
     return False
 
 
-def have_internet(ip_to_ping="1.1.1.1") -> bool:
```

Comment:
Any major reason why the switch from CloudFlare to Google?

Reply:
Not really, I just always default to Google and defaulted to it while rewriting the method. I can change it back to CloudFlare, no biggie

Reply:
To be completely honest I don't have an issue with Google here, was just curious!
In my personal experience I've had less outages with CloudFlare's DNS servers than Google's, but they're still 97% vs 98% 😛

Reply:
![grafik](https://user-images.githubusercontent.com/60011425/233654365-18beb3d3-1989-42fa-a5ea-92d6a0946488.png)
I concede my point to you this time, we save 0.02 seconds with CloudFlare